pkgs/development/tools/coursier/default.nix

```diff
@@ -33,6 +38,10 @@ stdenv.mkDerivation rec {
     fi
   '';
 
+  postInstall = ''
+    wrapProgram $out/bin/cs --prefix "LD_LIBRARY_PATH" : "${libPath}"
```

Comment:
Would you please test if adding `autoPatchelfHook` to `nativeBuildInputs` works?

Reply:
Wait, I can't believe `autoPatchelfHook` can work. I'm going back to your previous commit.